match/lib/match/storage/google_cloud_storage.rb

```diff
@@ -197,6 +197,10 @@ def ensure_keys_file_exists(google_cloud_keys_file)
           UI.input("Confirm with enter")
         end
 
+        UI.important("Please never add the #{DEFAULT_KEYS_FILE_NAME.cyan} file in version control.")
+        UI.important("Instead please add the file to your .gitignore")
+        UI.input("Confirm with enter")
```

Comment:
```suggestion
        UI.input("Confirm with [Enter]")
```

Reply:
Ah, just saw that this is used multiple times above... then probably separate PR worthy.